src/client/components/chart-data-buttons/chartbar-buttons/chart-data-button.component.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import './chart-data-button.style.css';
+import PropTypes from 'prop-types';
+import Button from '../../button/button.component';
+
+const ChartbarMenu = ({ buttons, defaultSelection }) => {
+  const [selectedButton, setSelectedButton] = useState(defaultSelection.id);
```

Comment:
I would change this a bit so we don't confuse it. It actually stores the id, not the button. This is a bit more clear ⬇️ 
```suggestion
  const [selectedButtonId, setSelectedButtonId] = useState(defaultSelection.id);
```

Reply:
@mamathamereddy this is an old PR that's waiting for some changes. Could you or someone on your team implement the feedback, and merge?